src/libraries/System.Runtime.InteropServices.JavaScript/gen/JSImportGenerator/JSImportStubContext.cs

```diff
@@ -118,12 +127,11 @@ static bool IsSkipLocalsInitAttribute(AttributeData a)
             };
             int typesHash = Math.Abs((int)hash);
 
-
-
             var fullName = $"{method.ContainingType.ToDisplayString()}.{method.Name}";
             string qualifiedName;
+
             var ns = string.Join(".", method.ContainingType.ToDisplayParts().Where(p => p.Kind == SymbolDisplayPartKind.NamespaceName).Select(x => x.ToString()).ToArray());
```

Comment:
Keep `method.ContainingType.ToDisplayParts()` in a local since it doesn't seem like a light weight method.